requirements/test.txt

```diff
@@ -4,3 +4,4 @@ pytest
 pytest-django
 pytest-cov
 coverage
+tox
```

Comment:
I was under the impression that you don't put tox as a test requirement because tox doesn't need tox in the test environments.